typescript/src/generator.ts

```diff
@@ -181,15 +181,12 @@ export class Generator {
   }
 
   private buildAPIObject(): API {
-    const protoFilesToGenerate = this.request.protoFile.filter(
-      pf =>
-        pf.name &&
-        this.request.fileToGenerate.includes(pf.name) &&
-        !API.isIgnoredService(pf)
-    );
-    const packageNamesToGenerate = protoFilesToGenerate.map(
-      pf => pf.package || ''
-    );
+    const protoPackagesToGenerate = new Set<string>();
```

Comment:
This fix follows https://github.com/googleapis/gapic-generator-typescript/pull/832 - this is just one more place I forgot to fix (and did not catch it because of lack of test coverage).  Now this is covered by tests :)